code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -1104,20 +1104,35 @@
 
 /obj/item/clothing/head/helmet/space/hardsuit/powerarmor_advanced
 	name = "Advanced power helmet"
-	desc = "It's an advanced power armor Mk I helmet. It looks somewhat threatening."
+	desc = "It's an advanced power armor X-01 helmet. It looks somewhat threatening."
 	icon_state = "hardsuit0-advpa1"
 	item_state = "advpa1_helm"
 	hardsuit_type = "advpa1"
-	armor = list(MELEE = 50, BULLET = 48, LASER = 25, ENERGY = 25, BOMB = 48, BIO = 0, RAD = 0, FIRE = 0, ACID = 25)
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/resonator, /obj/item/mining_scanner, /obj/item/t_scanner/adv_mining_scanner, /obj/item/gun/energy/kinetic_accelerator, /obj/item/twohanded/required/kinetic_crusher)
```

Comment:
```suggestion
```

Reply:
to clarify are you requesting I remove the suit slot adjustments?
This armor is meant for miners, and if I didn't add those, I'd have to make it a subtype of the mining hardsuit specifically. Additionally, the crusher was meant to make it somewhat unique, as currently there's not many ways to store one off-hand other than just dropping it and dragging it around. I do still need to dial in armor values so it doesn't wind up just being Drake Armor 2, in addition to that, however.

Reply:
OH WAIT
LMao I just noticed this is on the helmet.

Reply:
Alright it's fixed